src/cordial_gui/scripts/gui_controller.py

```diff
@@ -11,12 +11,12 @@
 class GuiController:
     _NODE_NAME = "gui_controller"
 
-    _MOUSE_EVENT_TOPIC = "cordial/gui/event/mouse"
-    _KEYPRESS_EVENT_TOPIC = "cordial/gui/event/keypress"
-    _NEW_SERVER_EVENT_TOPIC = "cordial/gui/event/new_server"
-    _USER_RESPONSE_TOPIC = "cordial/gui/user_response"
-    _DISPLAY_TOPIC = "cordial/gui/display"
-    _USER_PROMPTED_TOPIC = "cordial/gui/prompt"
+    _MOUSE_EVENT_TOPIC = rospy.get_param("_MOUSE_EVENT_TOPIC")
```

Comment:
Most of these parameters are used just once, it would be nice to get the parameter just where it's needed.

Same for other files.